### PR TITLE
fix dir bug on windows

### PR DESCRIPTION
**Pull request by daogangtang**
Usually we render template like 
```
render("foo/bar.html", Context::new())
```
we use '/', but on windows, tera generate dir file system hash with format like "foo\\bar.html".

This will meet panic when execute the render command. This pr solve this.

### CODE REVIEW

**Comment by daogangtang** [comment]
how to rebase these submission?

File: src/tera.rs [hunk 1]
```diff
@@ -49,13 +51,22 @@ impl Tera {
...
+        // println!("templates {:?}", templates);
+
         Tera {
             templates: templates
         }
     }
 
     pub fn render(&self, template_name: &str, data: Context) -> TeraResult<String> {
-        let template = self.templates.get(template_name).unwrap(); // TODO error handling
+        //let template = self.templates.get(template_name).unwrap(); // TODO error handling
```
Comment (Keats) [inline src/tera.rs]: you can remove that line as well

File: src/tera.rs [hunk 2]
```diff
@@ -49,13 +51,22 @@ impl Tera {
...
+        let template = match self.templates.get(template_name) {
+            Some(tmpl) => tmpl,
+            None => {
+                println!("error in render {}", template_name);
```
Comment (Keats) [inline src/tera.rs]: and that one as well

**Comment by Keats** [comment]
Do something like http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html for squashing commits

**Comment by Keats** [comment]
Sorry, ended up pushing the fix myself (https://github.com/Keats/tera/commit/32126ba735d9c9357ae6f8be6e014acdf1867942)

Thanks for the report!

**Comment by Keats** [comment]
@daogangtang can you try on master see if it works for you?

**Comment by daogangtang** [comment]
all right now.

**Pull request closed without merging**